tests/transports-integrations/tests/utils/common.py

```
@@ -89,6 +89,21 @@ class Config:
     {"role": "user", "content": "What's the weather in New York and calculate 15 * 23?"}
 ]
 
+# Streaming Test Messages
+STREAMING_CHAT_MESSAGES = [
+    {
+        "role": "user",
+        "content": "Tell me a short story about a robot learning to paint. Keep it under 200 words.",
+    }
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)